app/mailers/user_mailer.rb

```diff
@@ -449,7 +438,7 @@ class DueIssuesReminder
   def initialize(days = nil, project_id = nil, type_id = nil, user_ids = [])
     @days     = days ? days.to_i : 7
     @project  = Project.find_by_id(project_id)
-    @type  = Type.find_by_id(type_id)
+    @type  = ::Type.find_by_id(type_id)
```

Comment:
> In this case I agree with hound, as the multi spacing does not lead to harmonized alignment.

The alignment issue was there before. I haven't gone out of my way to clean up syntax in this PR.

> P.S.: Is it only my browser that is so utterly slow on this PR? ^^

Mine is dreadfully slow too.